src/etc/inc/pfsense-utils.inc

```diff
@@ -3574,4 +3574,19 @@ function is_url_hostname_resolvable($url) {
 	return $resolvable;
 }
 
+function get_pf_timeouts () {
```

Comment:
Since you are providing a function for this, it makes more sense for its output to be structured as `$pftimeout[<protocol>][<type>] = <value>` so whatever ends up calling this doesn't have to remember what is in index 1/2/etc.

Reply:
Updated -
now it returns array `$pftimeout[$proto][$type][name/keyname/value]`